Makefile

```diff
@@ -151,3 +151,11 @@ fix-abci-app-specs:
 	python -m autonomy.cli analyse abci generate-app-specs packages.valory.skills.oracle_deployment_abci.rounds.OracleDeploymentAbciApp packages/valory/skills/oracle_deployment_abci/fsm_specification.yaml || (echo "Failed to check oracle_deployment_abci consistency" && exit 1)
 	python -m autonomy.cli analyse abci generate-app-specs packages.valory.skills.price_estimation_abci.rounds.PriceAggregationAbciApp packages/valory/skills/price_estimation_abci/fsm_specification.yaml || (echo "Failed to check price_estimation_abci consistency" && exit 1)
 	echo "Successfully validated abcis!"
+
+AEA_AGENT_PRICE_ORACLE:=valory/oracle:latest:$(shell cat packages/packages.json | grep "agent/valory/oracle" | cut -d "\"" -f4 )
```

Comment:
would this cause problems in windows?

Reply:
Probably, `cat` is not available on Windows for example. But there are other commands in our `Makefile` which are not compatible with windows as well.

Reply:
so in my windows machine it fails with: 
```
'cat' is not recognized as an internal or external command, operable program or batch file. 
```
Also because this is not part of the `release-image` target, it gets evaluated on any make target. 
Maybe add it as part of the `release-image` target?, It will still not work on windows, but at least it won't interfere with other targets.

Reply:
Refactored in 5bdb6a0.